docs/versioning.md

```diff
@@ -0,0 +1,136 @@
+# Electron Versioning Document
+
+## Version 1.x
+
+Electron versions *< 2.0* have not conformed to the [semver](http://semver.org) spec. Major versions corresponded to end-user API changes. Minor versions corresponded to Chromium major releases. Patch versions corresponded to new features and bug fixes. While convenient for developers merging features, it creates problems for developers of client-facing applications. The QA testing cycles of major apps like Slack, Stride, Teams, Skype, VS Code, Atom, and Desktop can be lengthy and stability is a highly desired outcome. There is a high risk in adopting new features while trying to absorb bug fixes.
+
+Here is an example of the 1.x strategy:
+
+![](versioning/sketch-0.png)
+
+An app developed with `1.8.1` cannot take the `1.8.3` bug fix without either absorbing the `1.8.2` feature, or by backporting the fix and maintaining a new release line.
```

Comment:
> An app developed with `1.8.1` cannot take the `1.8.3` bug fix without either absorbing the `1.8.2` feature

But 1.8.1 -> 1.8.2 would necessarily only be a bug fix because it's a patch-level change, right?

Reply:
Not for the 1.x line. That only applies 2.x onward.